pages/system/modals.js

```diff
@@ -0,0 +1,237 @@
+import * as React from "react";
+import * as System from "~/components/system";
+import Group from "~/components/system/Group";
```

Comment:
Nit: This isn't a big deal, but lets keep imports grouped by style for less inference pain when you first read a file.